layouts/about-release-schedule.hbs

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="{{site.locale}}" {{#if site.rtl}}dir="rtl"{{/if}}>
+{{> html-head }}
+
+<body>
+  {{> header }}
+
+  <div id="main">
+    <div class="container has-side-nav">
+
+      {{> navigation key='about'}}
+
+      <article>
+        <h1>{{ title }}</h1>
+
+        <p>
+          <img alt="{{ title }}" src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.png">
```

Comment:
FYI the PNG might get replaced by SVG (https://github.com/nodejs/Release/pull/380). 

The image is generated from `schedule.json` (see commit message in https://github.com/nodejs/Release/commit/e82fc16b222555415948abdf728a8ee8f304dced). Would it make sense to generate it in the deploy step?

Reply:
As soon as the PR is merged, I'll switch to the SVG instead.

IMHO it doesn't make sense to be generated on each website update. The release schedule only changes only every couple of months and is updated first in https://github.com/nodejs/Release/ anyway.